scripts/validation/service.schema.js

```diff
@@ -1,12 +1,4 @@
-import { createRequire } from 'module';
-import path from 'path';
-import { fileURLToPath } from 'url';
-
-import config from 'config';
-
-const require = createRequire(import.meta.url);
-const __dirname = path.dirname(fileURLToPath(import.meta.url));
-const TYPES = require(path.resolve(__dirname, '../../', config.get('services.documentTypesPath'))); // eslint-disable-line import/no-dynamic-require
+import { DOCUMENT_TYPES as TYPES } from '../../src/archivist/services/index.js';
```

Comment:
not sure what benefit there is to `as TYPES`

maybe 
```
import { DOCUMENT_TYPES } from '../../src/archivist/services/index.js';
const AVAILABLE_TYPES_NAME = Object.keys(DOCUMENT_TYPES);
```

Not mandatory of course